[telemetrySettting] Create sampled Logger (#8134)
open-telemetry/opentelemetry-collector: exporter/exporterhelper/README.md

      [the batch processor](https://github.com/open-telemetry/opentelemetry-collector/tree/main/processor/batchprocessor)
      is used, the metric `batch_send_size` can be used for estimation)
- `timeout` (default = 5s): Time to wait per individual attempt to send data to a backend

> - `sampled_logger`: Enables/disables sampling of logging messages.

Reviewer: It is not clear what does enabling/disabling sampling do. Please expand the description.

Author: @tigrannajaryan thank you for your feedback.
Is it now more clear? Am I missing something?

Thanks

> - `sampled_logger`: Samples logging messages, which caps the CPU and I/O load of logging while keeping a representative subset of your logs.

Reviewer 2: ```suggestion
- `sampled_logger`: Enables/disables sampling of logging messages, which caps the CPU and I/O load of logging while keeping a representative subset of your logs.
```

> - `sampled_logger`: Enables/disables sampling of logging messages, which caps the CPU and I/O load of logging while keeping a representative subset of your logs.

Reviewer 3: ```suggestion
- `sampled_logger`: Enables/disables sampling of the log messages.
```

we could also replace `Enables/disables` by `Toggles`

Author: Keeping `Enables/disables ` for the moment
